lib/plugins/aws/provider.js

```diff
@@ -864,6 +864,7 @@ class AwsProvider {
                             uri: { $ref: '#/definitions/ecrImageUri' },
                             path: { type: 'string' },
                             file: { type: 'string' },
+                            buildArgs: { type: 'object' },
```

Comment:
Let's define that we expect string properties in that object, additionally, we should also error out when someone tries to define `buildArgs` on an image defined with `uri`. This check should be done not on schema level but instead in a similar way to these checks: https://github.com/serverless/serverless/blob/7601e26360226e77c4c8d631227a248e7a81afc2/lib/plugins/aws/provider.js#L1972